superset/charts/api.py

```diff
@@ -549,13 +549,18 @@ def data(self) -> Response:
             500:
               $ref: '#/components/responses/500'
         """
+        json_body = None
         if request.is_json:
             json_body = request.json
         elif request.form.get("form_data"):
             # CSV export submits regular form data
-            json_body = json.loads(request.form["form_data"])
-        else:
-            return self.response_400(message="Request is not JSON")
+            try:
```

Comment:
note: we should eventually refactor the frontend CSV export to use the expected payload